File: C/9498.cpp
# include <stdio.h>

int main()
{
	
// [0] 변수 선언부 
	int score;
	
// [1] 값을 입력받음 
	scanf("%d", &score);				

// [2] 조건에 맞게 값을 출력함 
	if (score >= 90)
	{
	    printf("A");
	}

	else if (score >= 80)
	{
    	printf("B");
	}
	else if (score >= 70)
	{
    	printf("C");
	}
	else if (score >= 60)
	{
    	printf("D");
	}
	else
	{
    	printf("F");
	}	
	
	return 0;
}

File: Python/9498.py
# [0] 성적 값을 입력 받음
score = int(input())

# [1] 조건에 맞게 값을 출력함
if (score >= 90):
    print("A")
elif (score >= 80):
    print("B")
elif (score >= 70):
    print("C")
elif (score >= 60):
    print("D")
else:
    print("F")
